dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -374,6 +380,16 @@ private void handlePotentialFormatEntry(Set<FileFormat> fileFormats, Object v) {
         }
     }
 
+    private Map<String, Object> convertToolToSingleStepWorkflow(Map<String, Object> tool) {
+        Map<String, Object> workflow = new HashMap<>();
+        workflow.put("cwlVersion", "v1.2");
```

Comment:
What happens if the tool you are wrapping is not 1.2? Presumably nothing downstream cares, but just double-checking.

Reply:
The `CommandLineTool` has its own version, which will take precedence when it is processed.

Reply:
Actually, the processing of the `CommandLineTool` doesn't care about the version.